src/components/Barrier.jsx

```diff
@@ -27,6 +28,28 @@ const Barrier = ({
     </div>
 ));
 
+Barrier.propTypes = {
```

Comment:
There are lots of space between property and value in all files, I'm not sure why we need them